src/strategies/information.rs

```diff
@@ -113,14 +119,72 @@ impl Question for IsPlayable {
         let ref mut card_table = hand_info[self.index];
         let possible = card_table.get_possibilities();
         for card in &possible {
-            if view.get_board().is_playable(card) {
+            if (self.property)(view.get_board(), card) {
                 if answer == 0 { card_table.mark_false(card); }
             } else {
                 if answer == 1 { card_table.mark_false(card); }
             }
         }
     }
 }
+fn q_is_playable(index: usize) -> CardHasProperty<impl Fn(&BoardState, &Card) -> bool> {
+    CardHasProperty {index, property: |board, card| board.is_playable(card)}
+}
+fn q_is_dead(index: usize) -> CardHasProperty<impl Fn(&BoardState, &Card) -> bool> {
+    CardHasProperty {index, property: |board, card| board.is_dead(card)}
+}
+
+struct AdditiveComboQuestion {
+    /// For some list of questions l, the question `AdditiveComboQuestion { questions : l }` asks:
+    /// "What is the first question in the list `l` that has a nonzero answer, and what is its
+    /// answer?"
+    /// If all questions in `l` have the answer `0`, this question has the answer `0` as well.
+    ///
+    /// It's named that way because the `info_amount` grows additively with the `info_amount`s of
```

Comment:
interesting - if some answer is non-zero then the form of the information is a bit weird.  if this is meant to be for only binary questions, and only useful to distinguish 0 vs. nonzero, then it seems good to save the information and coalesce all the nonzero cases, right?

Reply:
Yeah I can't think of a way to use this for questions that aren't binary. But theoretically one totally could! If we did, I would argue for the current semantics given that
* It feels like a "meaningful combination operation" given that `AdditiveComboQuestion(a, b, c) == AdditiveComboQuestion(AdditiveComboQuestion(a, b), c) == AdditiveComboQuestion(a, AdditiveComboQuestion(b, c))`
* I don't see how coalescing the nonzero cases is even doable, given that there's no way to call `acknowledge_answer` on a component question with just the information that the answer is nonzero.

Reply:
hey, i totally misunderstood earlier today.  you're right that acknowledge_answer is problematic, sorry!  

I'm also ashamed of myself for calling something with the units of e^nats "info_amount"